contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -1205,6 +1223,66 @@ pltsql_bbfViewHasInsteadofTrigger(Relation view, CmdType event)
 	return false;
 }
 
+/*
+ * get_func_id_from_exp
+ *  For expressions T_FuncExpr, T_OpExpr, T_Aggref returns its function Oid
+ */
+static Oid
+get_func_id_from_exp(Node *expr)
```

Comment:
Suggestion: 
```
bool
check_functions_in_node(Node *node, check_function_callback checker,
						void *context)
```

Can we use this ?

Reply:
Updated the logic to use above suggested function.